diff -dur psycopg-1.1.11.orig/aclocal.m4 psycopg-1.1.11/aclocal.m4
--- psycopg-1.1.11.orig/aclocal.m4	2003-12-18 23:08:23.000000000 +0100
+++ psycopg-1.1.11/aclocal.m4	2004-02-18 11:17:39.919319805 +0100
@@ -192,8 +192,8 @@
 AC_DEFUN(PY_CHECK_INSTALL,
 [AC_REQUIRE([PY_PYTHON_VERSION])
 AC_REQUIRE([PY_PYTHON_EXEC_PREFIX])
-if test -x "$PYTHON_EXEC_PREFIX/lib/python$PYTHON_VERSION/config/install-sh"; then
-  INSTALL="$PYTHON_EXEC_PREFIX/lib/python$PYTHON_VERSION/config/install-sh"
+if test -x "$PYTHON_EXEC_PREFIX/lib64/python$PYTHON_VERSION/config/install-sh"; then
+  INSTALL="$PYTHON_EXEC_PREFIX/lib64/python$PYTHON_VERSION/config/install-sh"
   AC_SUBST(INSTALL)
 else
   AC_PATH_PROG(INSTALL, install, install)
@@ -212,8 +212,8 @@
 sp = list(())
 for p in sys.path:
   sp.append(os.path.normpath(p))
-for dir in ('/lib/site-python', '/lib/python${PYTHON_VERSION}/site-packages',
-      '/lib/python${PYTHON_VERSION}'):
+for dir in ('/lib64/site-python', '/lib64/python${PYTHON_VERSION}/site-packages',
+      '/lib64/python${PYTHON_VERSION}'):
   p = os.path.normpath(sys.prefix+dir)
   if os.path.isdir(p) and p in sp:
     print dir
